test/controllers/csv_exports_controller_test.rb

```diff
@@ -76,16 +76,87 @@
     end
 
     describe "#new" do
-      before do
-        p = projects(:test).dup
-        p.name = "Other"
-        p.save!(validate: false)
+      describe "as html" do
+        before do
+          p = projects(:test).dup
```

Comment:
This was to create a second project to see if it appears in the dropdown.  I had a previous where it only showed one project in the UI.  It may be overkill to test UI logic.